assets/js/base/hooks/cart/use-store-cart.ts

```diff
@@ -7,9 +7,19 @@ import { CART_STORE_KEY as storeKey } from '@woocommerce/block-data';
 import { useSelect } from '@wordpress/data';
 import { useEditorContext } from '@woocommerce/base-context';
 import { decodeEntities } from '@wordpress/html-entities';
-import { mapValues } from 'lodash';
+import type {
+	StoreCart,
+	CartResponseTotals,
+	CartResponseFeeItem,
+	CartResponseBillingAddress,
+	CartResponseShippingAddress,
+} from '@woocommerce/types';
 
-const defaultShippingAddress = {
+declare module '@wordpress/html-entities' {
+	// eslint-disable-next-line no-shadow
```

Comment:
is this still needed now it's configured in eslint config?

Reply:
As of when? I can see it being turned off only for mapped file.